proxy/api/src/http/identity.rs

```diff
@@ -68,6 +80,18 @@ mod handler {
         Ok(reply::with_status(reply::json(&id), StatusCode::CREATED))
     }
 
+    /// Update the [`identity::Identity`] metadata.
+    pub async fn update(
+        ctx: context::Unsealed,
+        metadata: identity::Metadata,
+    ) -> Result<impl Reply, Rejection> {
+        session::get_current(&ctx.store)?.ok_or(http::error::Routing::NoSession)?;
+        let id = identity::update(&ctx.peer, metadata).await?;
+        session::update_identity(&ctx.store, id);
```

Comment:
FYI @FintanH 

So I've added this line calling the - also added - `session::update_identity` method because I started building the call from the UI calling this endpoint and noticed the session would never update.